scripts/build

```diff
@@ -115,44 +139,66 @@ cleanup() {
 }
 
 INSTALLBIN=$(pwd)/bin
-if [ ! -x ${INSTALLBIN}/cni ]; then
+PLUGINS_WINDOWS_PATCH=$(pwd)/scripts/plugins-windows.patch
```

Comment:
We already have a rancher fork of containernetworking/plugins at https://github.com/rancher/plugins.git - can you PR your patch into that repo, instead of checking the patch to this repo and applying it manually?

Reply:
@brandond a quick question, what is the main branch for the plugins repository now? I saw the current plugins version K3s using is `v1.2.0-k3s1` but it is a tag so I cannot create pull request over it.

Reply:
@manuelbuil ^ could you answer Sean's question here?

Reply:
Yep, I admit the ways of working on that project are a bit strange. IIRC, when there is a new version, we checkout the upstream branch and then cherry-pick our patches and create the tag. Let me create a new branch named `k3s-v1.3.0` so that we can work on that

Reply:
https://github.com/rancher/plugins/tree/k3s-v1.3.0 and it is based on https://github.com/containernetworking/plugins/tree/v1.3.0 and contains our patches

Reply:
@manuelbuil Thanks for the pointer! I created a pull request https://github.com/rancher/plugins/pull/7 and please let me know any feedback there.

And once the plugins pull request is merged, what's the next step? Would you be updating the K3s master branch to consume the new k3s-1.3.0 and then I come back to iterate on this k3s pull request? Let me know what's the plan and how I can proceed. Thanks!

Reply:
I've left some feedback on the PR; its very minor but once that's addressed we can merge that and then you can update your PR to pull in the new tag of the plugins in the build script.